File: RepoTransBench/target_projects/Java/Go/knightliao_pikaQ/tests/original/columns_test.go
package original

import "testing"

const (
	NAME       = "name"
	CAMPAIGN_ID = "campaignId"
)

func TestNameConstant(t *testing.T) {
	if NAME != "name" {
		t.Errorf("NAME = %s, expected name", NAME)
	}
}

func TestCampaignIdConstant(t *testing.T) {
	if CAMPAIGN_ID != "campaignId" {
		t.Errorf("CAMPAIGN_ID = %s, expected campaignId", CAMPAIGN_ID)
	}
}

File: RepoTransBench/target_projects/Java/C#/knightliao_pikaQ/tests/original/ColumnsTest.cs
using Xunit;

namespace KnightliaoPikaQ.Tests.Original
{
    // Simulate Columns class with constants
    public static class Columns
    {
        public const string NAME = "name";
        public const string CAMPAIGN_ID = "campaignId";
    }

    public class ColumnsTest
    {
        [Fact]
        public void TestNameConstant()
        {
            Assert.Equal("name", Columns.NAME);
        }

        [Fact]
        public void TestCampaignIdConstant()
        {
            Assert.Equal("campaignId", Columns.CAMPAIGN_ID);
        }
    }
}